tools/@aws-cdk/spec2cdk/temporary-schemas/us-east-1/aws-dynamodb-globaltable.json

```diff
@@ -0,0 +1,643 @@
+{
+    "typeName" : "AWS::DynamoDB::GlobalTable",
+    "description" : "Version: None. Resource Type definition for AWS::DynamoDB::GlobalTable",
+    "additionalProperties" : false,
+    "nonPublicProperties": [
+      "/properties/WarmThroughput",
+      "/properties/GlobalSecondaryIndexes/items/properties/WarmThroughput"
+    ],
+    "nonPublicDefinitions": [
+      "/definitions/WarmThroughput",
+      "/definitions/GlobalSecondaryIndex/properties/WarmThroughput"
+    ],
```

Comment:
We generally don't want temporary schemas to be merged in. This is more testing purposes on your local. Curious when will the schema be available and when's the expected launch date?